model/profile.py

```diff
@@ -0,0 +1,63 @@
+from flask import Blueprint, request
+
+from mongo import User
+
+from .utils import HTTPResponse, HTTPError, Request
+from .auth import login_required
+
+profile_api = Blueprint('profile_api', __name__)
+
+
+@profile_api.route('/<username>', methods=['GET'])
+@login_required
+def view_others_profile(user, username):
+    try:
+        user = User(username)
+        data = {
+            'username': user.username,
+            'email': user.obj.email,
+            'displayed_name': user.obj.profile.displayed_name,
+            'bio': user.obj.profile.bio
+        }
+    except:
+        return HTTPError('Profile not exist.', 404)
+
+    return HTTPResponse('Profile exist.', data=data)
+
+
+@profile_api.route('', methods=['GET', 'POST'])
+@login_required
+def view_or_edit_profile(user):
+    @Request.json(['displayed_name', 'bio'])
```

Comment:
```suggestion
    @Request.json(['bio'], vars_dict={'displayed_name': 'displayedName'})
```
JSON 的欄位名稱應該是 `displayedName`

Reply:
這個我有點看不懂...

Reply:
我也是，為何要這樣寫？

Reply:
為何要這樣寫？

Reply:
其他的我改了  剩這個

Reply:
那我就先merge了...

Reply:
傳過來的 JSON 的欄位名稱是 `displayedName`
要透過 Request.json 拿到其內容，並放進 `desplayed_name` 變數裡

Reply:
改了～